genet/use/road_pricing.py

```diff
@@ -1,27 +1,99 @@
-import os
 import json
 import logging
+import os
+
+import numpy as np
 import pandas as pd
 from lxml import etree as et
 from lxml.etree import Element, SubElement, Comment
 from tqdm import tqdm
 
 
-def extract_network_id_from_osm_csv(network, attribute_name, osm_csv_path, outpath):
+class Cordon:
+    def __init__(self, df_tolls: pd.DataFrame = None):
+        if df_tolls is None:
+            self.tolls_df = pd.DataFrame(
```

Comment:
Are we deliberately varying the member name here (`df_tolls` v `tolls_df` on the other side of the `if` clause at line 28)? I can see from the coverage report that line 15 is not tested, so it makes me think maybe this is a typo?

Reply:
oops, typo!